header.php

```diff
@@ -1,422 +1,423 @@
+                            <?php if($twitter = dsi_get_option( "twitter", "socials" )) :?><a href="<?php echo $twitter; ?>" aria-label="twitter" title="vai alla pagina twitter"><svg class="icon it-social-twitter"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#it-social-twitter"></use></svg></a><?php endif; ?>
+                            <?php if($linkedin = dsi_get_option( "linkedin", "socials" )) :?><a href="<?php echo $linkedin; ?>" aria-label="linkedin" title="vai alla pagina linkedin"><svg class="icon it-social-linkedin"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#it-social-linkedin"></use></svg></a><?php endif; ?>
+                        </div><!-- /header-social-wrapper -->
+                    </div><!-- /header-social -->
+                    <?php endif ?>
+                </div><!-- /col -->
+            </div><!-- /row -->
+        </div><!-- /container -->
+
+        <div class="bg-white d-none d-xl-block header-bottom" id="sub-nav">
+            <div class="container">
+                <div class="row variable-gutters">
+                    <div class="col nav-container">
+                        <nav aria-label="Principale" class="main-nav" id="menu-principale">
+                            <ul class="dl-menu nav-list nav-list-primary" data-element="menu">
+                                <?php
+                                // check if scuola has menu
+                                $theme_location = "menu-scuola";
+                                $option_location = "item_menu_scuola";
+                                unset($menu_obj);
+                                if(isset($theme_locations[$theme_location]))
+                                    $menu_obj = get_term( $theme_locations[$theme_location], 'nav_menu' );
+                                if(isset($menu_obj) && !is_wp_error($menu_obj)) {
+                                    $landing_url = dsi_get_template_page_url("page-templates/la-scuola.php");
+                                    if($landing_url)
+                                        $items_wrap = '<ul class="%2$s" data-element="school-submenu"><li class="menu-title" ><div class="h3"><a class="list-item" href="'.$landing_url.'">'.__("Panoramica", "design_scuole_italia").'</a></div></li>%3$s</ul>';
+                                    else
+                                        $items_wrap = '<ul class="%2$s" data-element="school-submenu">%3$s</ul>';
+
+                                    ?>
+                                    <li class="text-redbrown menu-dropdown-simple-wrapper">
+                                        <a class="nav-link dropdown-toggle <?php echo $active_page == 'school' ? 'active' : ''?>" data-toggle="dropdown"  role="button" href="#" aria-expanded="false" id="mainNavDropdown1_footer"><?php _e("Scuola","design_scuole_italia"); ?></a>

```

Comment:
cambiato in parametro ID il valore, aggiungendo il suffisso _footer e cosi per tutte le voci principali di menu nel footer del frontend